device/dell/x86_64-dell_s6100_c2538-r0/installer.conf

```diff
@@ -1,3 +1,3 @@
 CONSOLE_PORT=0x2f8
 CONSOLE_DEV=1
-ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="module_blacklist=gpio_ich"
+ONIE_PLATFORM_EXTRA_CMDLINE_LINUX="module_blacklist=gpio_ich nos-config-part=/dev/sda12"
```

Comment:
nos-config-part should be a separate variable.

Reply:
i think it is ok. not sure what is the benefit to having a separate variable. anyway, we need to support this variable for the fast reboot case.